.github/workflows/minor-schema-version-bump.yml

```diff
@@ -0,0 +1,211 @@
+      - name: Pull AWS Secrets
+        uses: aws-actions/aws-secretsmanager-get-secrets@v1
+        with:
+          # TODO: Change to prod and make separate, manual dev workflow once ready for deploy
+          secret-ids:
+            AUTH0_SECRETS, corpora/backend/dev/auth0-secret
+      - name: Check if any GENCODE csv files have changed
+        id: changed-files
+        uses: tj-actions/changed-files@v36
+        with:
+          files: cellxgene_schema_cli/cellxgene_schema/ontology_files/*.csv.gz
+      - name: Run Dry-Run Script
+        if: steps.changed-files.outputs.any_changed == 'true'
+        run: python3 -m schema_bump_dry_run_scripts.genes.gene_bump_dry_run
+        env:
+          corpus_env: "dev" # TODO: Change to prod and make separate, manual dev workflow once ready for deploy
+      - name: upload to slack
+        if: steps.changed-files.outputs.any_changed == 'true'
+        run: |
+          curl -F file=@genes-curator-report.txt -F "initial_comment=Gene Dry Run Report" -F channels=${{secrets
+          .SLACK_CURATOR_REPORTING_CHANNEL}} -H "Authorization: Bearer ${{secrets.SLACK_CURATOR_REPORTING_APP_AUTH}}" https://slack.com/api/files.upload
+
+  publish-to-pypi:
+    name: Build and publish Python distributions to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
+      - name: setup git
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+      - name: Bump and tag release version
+        run: |
+          pip install bumpversion
+          pip install wheel
+          bumpversion --config-file .bumpversion.cfg minor --allow-dirty
+          bumpversion --config-file .bumpversion.cfg prerel --allow-dirty --tag
+          git push origin --tags
+      - name: Build dist
+        run: >-
+          make pydist
+      - name: Publish distribution to Test PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository-url: https://test.pypi.org/legacy/
+          packages-dir: cellxgene_schema_cli/dist/
+      - name: Install and Test Package from Test PyPI
+        run: |
+          pip install --index-url https://test.pypi.org/simple/ cellxgene-schema
+          cellxgene-schema validate cellxgene_schema_cli/tests/fixtures/h5ads/example_valid.h5ad
+# TODO: Uncomment once ready to release to prod PyPI
+#      - name: Publish distribution to PyPI
+#        if: startsWith(github.ref, 'refs/tags')
+#        uses: pypa/gh-action-pypi-publish@release/v1
+#        with:
+#          password: ${{ secrets.PYPI_API_TOKEN }}
+#          package-dir: cellxgene_schema_cli/
+
+  generate-conversion-script:
+    runs-on: ubuntu-latest
+    needs: [ontology-dry-run, publish-to-pypi]
```

Comment:
this should only depend on publish-to-pypi

Reply:
it depends on ontology dry run to populate + write the map of deprecated terms with known replacement terms